templates/stream.html

```diff
@@ -1,9 +1,17 @@
 {% extends "base.html" %} {% block content %}
 
-<h1>socialDISTRIBUTION</h1>\
+<h1>socialDISTRIBUTION</h1>
 <div class="container">
 	<a href="{% url 'posts:new' %}">New post</a>
 
+	<ul>
+		{% for post in object_list %}
+			{% include "posts/partials/_stream_post.html" %}
+		{% empty %}
+			<li> No posts yet. </li>
+		{% endfor %}
+	</ul>
+
 </div>
 
-{% endblock %}
+{% endblock content %}
```

Comment:
Wait were we doing it wrong the whole time? hahahah

Reply:
I added an extension for formatting and it gave me an warning saying that the endblock should be named, both work but this is more explicit I guess?

Reply:
Oh sweet, good to know. Sounds good let's do this for everything else onwards. I'll update the remainings in a different PR